backend/src/main/java/com/benine/backend/http/PresetsRecallPresetHandler.java

```diff
@@ -0,0 +1,76 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.Preset;
+import com.benine.backend.PresetController;
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.Camera;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.CameraController;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import org.eclipse.jetty.server.Request;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class PresetsRecallPresetHandler extends RequestHandler {
```

Comment:
I think that the name of this class is a bit vague. Can this maybe be changed to something that's more clear?